deploy/11_obsolete/0064_deploy_old_catalyst_minter_with_220_legendary.ts

```diff
@@ -0,0 +1,138 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {BigNumber} from '@ethersproject/bignumber';
+import {parseEther} from '@ethersproject/units';
+
+const func: DeployFunction = async function (hre) {
+  const {deployments, getNamedAccounts} = hre;
+  const {deploy, read, execute, catchUnknownSigner} = deployments;
+
+  const {deployer, catalystMinterAdmin} = await getNamedAccounts();
+
+  const registry = await deployments.get('CatalystRegistry');
+  const sand = await deployments.get('Sand');
+  const asset = await deployments.get('Asset');
+  const gem = await deployments.get('Gem');
+  const catalyst = await deployments.get('Catalyst');
+
+  const bakedMintData = [];
+  for (let i = 0; i < 4; i++) {
+    const mintData = await read('Catalyst', 'getMintData', i);
+    const maxGems = BigNumber.from(mintData.maxGems).mul(
+      BigNumber.from(2).pow(240)
+    );
+    const minQuantity = BigNumber.from(mintData.minQuantity).mul(
+      BigNumber.from(2).pow(224)
+    );
+    let mintDataMaxQuantity = mintData.maxQuantity;
+    if (i == 3) {
+      if (mintDataMaxQuantity !== 200) {
+        throw new Error('was supposed to be 200');
+      }
+      mintDataMaxQuantity = 220;
+    }
+    const maxQuantity = BigNumber.from(mintDataMaxQuantity).mul(
+      BigNumber.from(2).pow(208)
+    );
+    const sandMintingFee = BigNumber.from(mintData.sandMintingFee).mul(
+      BigNumber.from(2).pow(120)
+    );
+    const sandUpdateFee = BigNumber.from(mintData.sandUpdateFee);
+    const bakedData = sandUpdateFee
+      .add(sandMintingFee)
+      .add(maxGems)
+      .add(minQuantity)
+      .add(maxQuantity);
+    bakedMintData.push(bakedData);
+  }
+
+  const catalystMinter = await deploy('CatalystMinterLegendary220', {
+    contract: 'CatalystMinter',
+    from: deployer,
+    log: true,
+    skipIfAlreadyDeployed: true,
+    args: [
+      registry.address,
+      sand.address,
+      asset.address,
+      gem.address,
+      sand.address,
+      catalystMinterAdmin,
+      '0x0000000000000000000000000000000000000000', // TODO SAND : mintingFeeCollector
+      parseEther('1'), //TODO SAND : confirm
```

Comment:
Hi @wighawag what's the note for re SAND?

Reply:
we currently do not have SAND enabled as fee. if we do, we need to specify a address to receive and decide on the fee